### ISSUE TITLE
Downloaded/uploaded files in job

### ISSUE THREAD

**Issue by odinuv** [keboola/kbc-ui#1593]
with new processor based components - s3ex, http ex - it's perfectly ok, to download only files and the job Storage Stats looks like it did nothing:

![image](https://user-images.githubusercontent.com/4319320/38041796-f5c6013c-32b2-11e8-8d96-395b3905e6bb.png)

**Comment by odinuv** [comment]
no, the point is that the files are not displayed there, so it seems no output was produced (unless you go through the events).  So it should not be hidden, it should say that some files were produced

**Comment by ujovlado** [comment]
Uvazujem nad tym, aku hodnotu to userovi prinesie. Lebo ak je to len o tom, dat mu info, ze to nieco urobilo, tak pred "Project Power" len hodme: "Processed files: 1" a tym je to odbavene -> bude vediet, ze sa tam nieco dialo.

![screenshot_2018-05-02_16-05-38](https://user-images.githubusercontent.com/419849/39527818-b376b2dc-4e22-11e8-98ee-1e16665517fc.png)

A to cislo by sme mohli ofiltrovat o spominane "table-import" tagy.
